packages/site/src/components/Navigation/TableOfContents.tsx

```diff
@@ -163,7 +163,7 @@ export const TableOfContents = ({
     <div
       ref={tocRef}
       className={classNames(
-        'fixed xl:article-grid article-grid-gap xl:w-screen xl:pointer-events-none overflow-auto max-xl:min-w-[300px]',
+        'fixed xl:article-grid article-grid-gap xl:w-screen xl:pointer-events-none overflow-auto max-xl:min-w-[300px] z-20',
```

Comment:
@rowanc1 thoughts on this? it was to accomodate the z-10 being added to the compute controls and to prevent overlap on narrow screens which made the first TOC item unclickable

Reply:
This will interfere with cross-reference popups.

Reply:
Can we back this out and put pointer-events none on the part of the control that isn't active?

Reply:
I have reverted. Given that the controls need another pass, I think we should have them not work rather than relying on the z-index pattern for the app (I had gotten away from that with the exception of the top-nav).